control_constants:
  update_rate: 100.0 # [Hz]   publish motor torques with this frequency. every 10ms = 100Hz

  # controller_type:        # "2D","3D","drive", this value is set through launch file!
  # motors_controller_type: # "PositionJointInterface", "VelocityJointInterface", "EffortJointInterface" this value is set through launch file.

  #Control Matrices for Pole Placement:
  R_  : [ 6.21845521e+01, 7.55026005e-04, 0.0, -0.00000000e+00, -0.00000000e+00, -0.00000000e+00 , -3.58060099e+00 , 4.59203044e+00,       -0.00000000e+00, -0.00000000e+00 , 3.07779971e+00 , 3.95866474e-01 ,-3.58060099e+00, -2.02082733e+00, -0.00000000e+00 ,-0.00000000e+00]
  F_  : [0.0, 3.03124099,    3.03124099, 0.0 ]
